### DISCUSSION TITLE
Amazon Marketing Stream with Ads Reports API

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#304] [Q&A] [answered]
Hi Everyone,

We're using the Amazon Ads Reports API to fetch data for the last 90 days and periodically re-request the API to get the most recent values. Recently, we've started using Amazon Marketing Stream (AMS) and noticed that AMS provides updated data that overlaps with what we receive from the Reports API. 

Is there a way to map the data from AMS to the data we get from the Reports API?

Thanks!

**Comment by kdwivedi13** [comment, marked as answer]
Hey Nilanth,

Thank you for your question regarding the mapping of data between the Amazon Ads Reports API and Amazon Marketing Stream (AMS). Currently, there is no direct one-to-one mapping available between the data provided by the Reports API and the data available in AMS. However, we can provide you with the following information, which will help you draw parallels between the two:

Reports available in the Amazon Ads Reports API:
https://advertising.amazon.com/API/docs/en-us/guides/reporting/v3/report-types/overview

Datasets available in Amazon Marketing Stream:
https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-stream/data-guide

Please note that Amazon Marketing Stream is a continuously evolving, and some datasets might be in closed beta or under development. If you are interested in accessing datasets that are currently in closed beta, kindly provide your email address or reach out to our support team for further assistance.

We understand the importance of having a seamless integration between different data sources, and our teams are working diligently to improve the mapping and compatibility between the Reports API and Amazon Marketing Stream. We appreciate your patience as we continue to enhance our services.

Thank you for reaching out and your engagement. If you have any further questions or require additional information, please do not hesitate to contact us.

Thanks.

**Reply by Nilanth** [reply]
Hi @kdwivedi13 

Thanks for the detailed reply. We are interested in accessing datasets that are currently in closed beta. My email Is support@kwickmetrics.com.
